tests/generic/vagga.yaml

```diff
@@ -307,6 +292,7 @@ commands:
 
   args: !Supervise
     description: Test of arg parsing
+    mode: wait-all-successful
```

Comment:
I'm not sure I understand. You use only `mode: wait-all-successful` in tests, actual we mainly run vagga in default mode.

So at a glance it looks like makes tests successful but doesn't test what was intended. Or am I missing something?

Reply:
Those tests are used to test other features. `wait-all-successful` just simplifies writing test cases.

Also there are tests with default mode.

Reply:
Okay. If you're sure about that, let's fix the quotes and we are good to merge.